Porting across bloom documentation (#2375)
ros2/ros2_documentation: source/How-To-Guides/Releasing/_Next-Steps.rst

Once your pull request has been submitted, usually within one or two days, one of the maintainers of rosdistro will review and merge your Pull Request.

> 24-48 hours after that, your packages will become available in the **ros-testing** repository, where you can :doc:`test your pre-release binaries <../../../Installation/Testing>`.

Reviewer: ```suggestion
If your package build is successful, in 24-48 hours your packages will become available in the **ros-testing** repository, where you can :doc:`test your pre-release binaries <../../../Installation/Testing>`.
```

> Approximately every two weeks, the distribution's release manager manually synchronizes the contents of ros-testing into the main ROS repository.

Reviewer: ```suggestion
Approximately every two to four weeks, the distribution's release manager manually synchronizes the contents of ros-testing into the main ROS repository.
```
